[BUG] syntax error near unexpected token `then'

I'm getting the following error when running with a standard prettier configuration
```
/home/runner/work/_actions/creyD/prettier_action/v4.1/entrypoint.sh: line 123: syntax error near unexpected token `then'
Error: Process completed with exit code 2.
```
Looks like the line was changed in commit dbdc55734ebeb55f608bde04f819938ecd8d906b any chance you can spot the error. It looks fine for me

Using the following .yml file:
```
name: Continuous Integration

on:
  pull_request:
    branches: [master]

jobs:
  prettier:
    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v2
        with:
          # Make sure the actual branch is checked out when running on pull requests
          ref: ${{ github.head_ref }}
          # This is important to fetch the changes to the previous commit
          fetch-depth: 0

      - name: Prettify code
        uses: creyD/prettier_action@v4.1
        with:
          prettier_version: '2.3.2'
          # This part is also where you can pass other options, for example:
          prettier_options: --write --config .prettierrc **/*.{js,jsx,less,ts,tsx}
          only_changed: True
```

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -113,6 +113,15 @@ if _git_changed; then
       git add "${INPUT_FILE_PATTERN}" || echo "Problem adding your files with pattern ${INPUT_FILE_PATTERN}"
     fi
 
+    # Check if the commit description was populated
+    IF_COMMIT_DESC=""
+    if [ "$INPUT_COMMIT_DESCRIPTION" != "" ]
+    then
+        $IF_COMMIT_DESC=" -m '$INPUT_COMMIT_DESCRIPTION'"
+    else
+        $IF_COMMIT_DESC=""
+    fi
+
     # Commit and push changes back
     if $INPUT_SAME_COMMIT; then
       echo "Amending the current commit..."
@@ -120,7 +129,7 @@ if _git_changed; then
       git commit --amend --no-edit
       git push origin -f
     else
-      git commit -m "$INPUT_COMMIT_MESSAGE" if [ "$INPUT_COMMIT_DESCRIPTION" != "" ]; then echo '-m "$INPUT_COMMIT_DESCRIPTION"' fi --author="$GITHUB_ACTOR <$GITHUB_ACTOR@users.noreply.github.com>" ${INPUT_COMMIT_OPTIONS:+"$INPUT_COMMIT_OPTIONS"} || echo "No files added to commit"
+      git commit -m "$INPUT_COMMIT_MESSAGE""$IF_COMMIT_DESC" --author="$GITHUB_ACTOR <$GITHUB_ACTOR@users.noreply.github.com>" ${INPUT_COMMIT_OPTIONS:+"$INPUT_COMMIT_OPTIONS"} || echo "No files added to commit"
       git push origin ${INPUT_PUSH_OPTIONS:-}
     fi
     echo "Changes pushed successfully."

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -117,9 +117,9 @@ if _git_changed; then
     IF_COMMIT_DESC=""
     if [ "$INPUT_COMMIT_DESCRIPTION" != "" ]
     then
-        $IF_COMMIT_DESC=" -m '$INPUT_COMMIT_DESCRIPTION'"
+        IF_COMMIT_DESC=" -m '$INPUT_COMMIT_DESCRIPTION'"
     else
-        $IF_COMMIT_DESC=""
+        IF_COMMIT_DESC=""
     fi
 
     # Commit and push changes back

